src/mlpack/methods/preprocess/preprocess_split_main.cpp

```diff
@@ -67,23 +67,21 @@ int main(int argc, char** argv)
   CLI::ParseCommandLine(argc, argv);
   const string inputFile = CLI::GetParam<string>("input_file");
   const string inputLabels = CLI::GetParam<string>("input_labels_file");
-  string trainingFile = CLI::GetParam<string>("training_file");
-  string testFile = CLI::GetParam<string>("test_file");
-  string trainingLabelsFile = CLI::GetParam<string>("training_labels_file");
-  string testLabelsFile = CLI::GetParam<string>("test_labels_file");
+  const string trainingFile = CLI::GetParam<string>("training_file");
+  const string testFile = CLI::GetParam<string>("test_file");
+  const string trainingLabelsFile = CLI::GetParam<string>("training_labels_file");
+  const string testLabelsFile = CLI::GetParam<string>("test_labels_file");
   const double testRatio = CLI::GetParam<double>("test_ratio");
 
   // Check on data parameters.
   if (trainingFile.empty())
```

Comment:
Shouldn't this now be `if (!CLI::HasParam("training_file"))` since there's now a default value? (same for the other parameters)